backend/migrations/000034_alter_communities_table.down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE communities DROP COLUMN contract_type;
```

Comment:
does the frontend ensure this value will never be `nil`? Always either nft or ft type?

Reply:
Yep - German commented on that. Default falls back to ft.